### PR TITLE
Update authorization step layout in onboarding flow

### PR DESCRIPTION
**Pull request by gtsiolis**
Minor UI updates on the authorization step layout in onboarding flow.

| BEFORE | AFTER |
|-|-|
| auth-before | auth-after |
| auth-dark-before | auth-dark-after |

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankoehnlein** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankoehnlein` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/projects/NewProject.tsx
Comment (gtsiolis) [inline components/dashboard/src/projects/NewProject.tsx:262]: **issue**: Prompting for selecting a different account here would make more sense once we add the provider dropdown selection above this area. See specs in https://github.com/gitpod-io/gitpod/issues/4948.

File: components/dashboard/src/projects/NewProject.tsx
Comment (gtsiolis) [inline components/dashboard/src/projects/NewProject.tsx:316]: **issue**: This is missing the stepper component as seen in the specs in https://github.com/gitpod-io/gitpod/issues/4948. ● ○ ○

**Comment by gtsiolis** [comment]
@jankeromnes could you take a took a this small change? 🏀 

/assign 
/cc @jankeromnes

**Comment by jankeromnes** [comment]
Hmm, the build seems to have failed:

```
Downloading rabbitmq from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Error: UPGRADE FAILED: another operation (install/upgrade/rollback) is in progress
```

Looks like a random intermittent problem. Re-triggering might fix.

/werft run

**Comment by jankeromnes** [comment]
Taking a look! 👓

**Review by jankeromnes — APPROVED** [review]
Nailed it! 🎯 🔨 🎉

110% perfect, nothing for me to complain of even comment about. 😁 Also this looks so much better now! 😍 ✨ 

[Insert George-vs-PR meme here]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: c0f2d743d6b1d66b27f60862ce33e24a31404eea</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5049#pullrequestreview-723296081" title="Approved">jankeromnes</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jankeromnes** [comment]
The secret password to make @roboquat happy here is:

/approve no-issue

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5049#issuecomment-893422219" title="Approved">jankeromnes</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5049#issuecomment-893422219" title="Approved">jankeromnes</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Thanks @jankeromnes! 🏀

**Pull request merged**
